_includes/tutorials/schedule-ktable-ttl/kstreams/code/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM confluentinc/cp-kafka-connect-base:5.5.1
```

Comment:
You can delete this since the tutorial doesn't use datagen or connect

Reply:
Deleting the Dockerfile